includes/AJAX.php

```diff
@@ -101,6 +107,104 @@ function( $name, $slug ) use ( $term ) {
 	}
 
 
+	/**
+	 * Syncs the changes with the Marketing Api for different ASC campaigns.
+	 *
+	 * Retrieves the changeset for each campaign type and posts them to the backend.
+	 * Makes sure that there is something to be sent.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return string
+	 */
+	public function publish_ad_changes() {
+
+		$asc_rt = 'retargeting';
```

Comment:
2 points here:

1. Can we use more descriptive variable names? eg: `retargeting` instead of `$rt`
2. `$asc_nb` and `$asc_rt` already exists as constants `NewBuyers::ID` and `Retargeting`. These are redeclared in a few places and not always consistently used.